### DISCUSSION TITLE
How to move the player database

### DISCUSSION THREAD

**Discussion by sRawiz** [Zrips/CMI#4267] [Q&A]
**Question: **
I changed the version of "CMI" from the old version to the new "CMI". How can I migrate the player database?

**Comment by CloudeLecaw** [comment]
> **Question: **
> I changed the version of "CMI" from the old version to the new "CMI". How can I migrate the player database?

what kind of "old version" u used before? - a bit more detail would be helüfull
Normally u dont need an extra migration process here so there is no migration for that.
if you updated from old CheckMyInventory to ContentManagementInterface  then the amount of before saved data would be really really small. i dont recomment this kind of migration.

When allready used ContentManagementInterface and your database is not currupted then all should work in todays database too. If you used mysql before - use same mysqldatabase, if you used the sqlite before copy the old file over the new (while server is offline)

_note:_
dont forget that players inventory, enderchest, stats and archivments saved in your main worldfolder of your server if you used no other plugin to hand over this before - cmi does not delete them - only when u enbaled this purge option on remove user or purge

**Comment by CloudeLecaw** [comment]
you also can import the playerdatas from your worldfolder with
`cmi importoldusers -> Imports users from playerdata folder in main world folder. Server can suffer lag spike during import`
but this does not import data from old cmi to new cmi :3

**Comment by mrfloris** [comment]
Upgrading from CMI version old to new is just a matter of replacing the jar file. There's nothing else to migrate or convert.

**Reply by CloudeLecaw** [reply]
Correct..well i thought he installed new cmi and started with a new database and wants to import now from the old file.

**Reply by mrfloris** [reply]
> Correct..well i thought he installed new cmi and started with a new database and wants to import now from the old file.

If it is a. "new" database on the new server, then just ditch it and use the old database. if you're importing it anyway .. the logic behind this makes no sense to me. sorry. 

new: 0 data
old: some data

i want some old data into my new database.. ---> use the old database.
